### ISSUE TITLE
Add template for empty sitelet project

### ISSUE THREAD

**Issue by granicz** [dotnet-websharper/templates#31]
It is quite often that one needs an empty sitelet to start from. Usually, I perform the following steps to get there (note how the project names tranfers as namespace in the code files):

1. `dotnet new -lang f# -n MyProject`

2. `dotnet add package WebSharper` and `dotnet add package WebSharper.AspNetCore`

3. Add a simple `Site.fs` as the first code file in the project, with:

    ```fsharp
    module Site

    open WebSharper
    open WebSharper.Sitelets

    [<Website>]
    let Main = Application.Text (fun ctx -> "Hello World!")
    ```

4. Merge `Program.fs` and `Startup.fs` into a single combined `Startup.fs`:

    ```fsharp
    namespace MyProject

    open System
    open Microsoft.AspNetCore
    open Microsoft.AspNetCore.Builder
    open Microsoft.AspNetCore.Hosting
    open Microsoft.AspNetCore.Http
    open Microsoft.Extensions.Configuration
    open Microsoft.Extensions.DependencyInjection
    open Microsoft.Extensions.Hosting
    open WebSharper.AspNetCore

    type Startup() =

        member this.ConfigureServices(services: IServiceCollection) =
            services.AddSitelet(Site.Main)
                .AddAuthentication("WebSharper")
                .AddCookie("WebSharper", fun options -> ())
            |> ignore

        member this.Configure(app: IApplicationBuilder, env: IWebHostEnvironment) =
            if env.IsDevelopment() then app.UseDeveloperExceptionPage() |> ignore

            app.UseAuthentication()
                .UseStaticFiles()
                .UseWebSharper()
                .Run(fun context ->
                    context.Response.StatusCode <- 404
                    context.Response.WriteAsync("Page not found"))

    module Program =
        let BuildWebHost args =
            WebHost
                .CreateDefaultBuilder(args)
                .UseStartup<Startup>()
                .Build()

        [<EntryPoint>]
        let main args =
            BuildWebHost(args).Run()
            0
    ```

I believe it would be a benefit to make this a dedicated template, say as `websharper-min`.

**Comment by granicz** [comment]
And I guess we need the equivalent C# template as well.

**Comment by Tarmil** [comment]
I think that would be nice to implement as an option on the `websharper-web` template, rather than yet another template. I did that for Bolero, `dotnet new bolero-app` creates a full-featured app and `dotnet new bolero-app --minimal` creates an empty app.

**Comment by granicz** [comment]
@Tarmil Frankly, I dare not touch the current logic of generating a NuGet+VSIX, or do you see an easy way to add this option? It would indeed be the best way forward, just not sure whether the whole repo needs a rewriting to enable it or not.

**Comment by Tarmil** [comment]
Hmm, I was only thinking about the dotnet cli template; I don't know how that translates for the VSIX.

**Comment by btamas2000** [comment]
Closed as solution #33 is merged to ws5
